### ISSUE TITLE
NULL inet and json types are not handled

### ISSUE THREAD

**Issue by kindermoumoute** [kyleconroy/sqlc#739]
Table is created with the following query:
```SQL
CREATE TABLE example (
    key uuid NOT NULL,
    my_ip inet
);
```

Generated query will be:
```SQL
-- name: CreateExample :one
INSERT INTO example (key) VALUES ($1) RETURNING *;
```

Then when I call the query I got this error when calling `CreateExample(ctx, uuid.New())`:
```
sql: Scan error on column index 1, name "my_ip": unsupported Scan, storing driver.Value type <nil> into type *net.IP
```

EDIT: replace `inet` with `json` and the same issue happens. More types might be involved.

**Comment by kindermoumoute** [comment]
Looks like we can't just use [golang standard types](https://github.com/kyleconroy/sqlc/blob/613de799c8133adb471e28b5297db20486e83a62/internal/codegen/golang/postgresql_type.go#L108-L109) (`net.IP`, `net.HardwareAddr`, `json.RawMessage`...) since they don't properly implement `Scan`. There are a bunch of opensource driver that already fixed this such as [pgtype](https://github.com/jackc/pgtype/blob/master/inet.go#L19-L22).

**Comment by kyleconroy** [comment]
The net.IP portion of this has been addressed in #1137. The JSON portion is a duplicate of #129. Don't worry though! The JSON side of things will also be fixed in v1.10.0
